crates/agglayer-config/src/prover.rs

```diff
@@ -95,6 +101,11 @@ impl Default for CpuProverConfig {
 pub struct NetworkProverConfig {
     #[serde(default = "default_activation_network_prover")]
     pub enabled: bool,
+
+    #[serde(default = "default_network_proving_task_timeout")]
+    #[serde_as(as = "DurationSeconds")]
+    pub proving_task_timeout: Duration,
```

Comment:
I'm not sure the name is very descriptive. Would something like `proving_request_timeout`  or `proof_handling_timeout` work?

Minor: Also, is it possible to make the default equal to `proving_timeout + 1`? I.e. if the user sets proving timeout to `t`, this becomes `t+1` unless explicitly specified.